### ISSUE TITLE
HD WORKS ON OTHER ADDONS 

### ISSUE THREAD

**Issue by cabban** [mrknow/filmkodi#392]
hello none tw show hd 720/1080, but other work perfectly
please fix best regards

**Comment by xxcriticxx** [comment]
ja tez to zauwazylem

**Comment by cabban** [comment]
Maybe write in English mate

**Comment by xxcriticxx** [comment]
i got same results as you no HD links for tv shows with latest version

**Comment by cabban** [comment]
Yes very annoying

**Comment by xxcriticxx** [comment]
we have to wait for mrknow maybe he will shop up today and do quick fix

**Comment by cabban** [comment]
Hope so mate :-)

**Comment by xX-Nexus-Xx** [comment]
I just raised another issue with TV-Shows in latest version .. its not streaming properly even for SD sources

**Comment by cabban** [comment]
no unfortunatly not :/ great addon, but if not working not great

**Comment by xxcriticxx** [comment]
HD is back by me

![capture](https://cloud.githubusercontent.com/assets/15007183/16763031/1ef1eda0-47f5-11e6-9265-973573337b19.PNG)

**Comment by cabban** [comment]
have u tested all the shows ?
ex baby daddy young and hungry etc can u test i have not muy computer atm :)

**Comment by xxcriticxx** [comment]
i am at work right now maybe tonight

**Comment by dracosc** [comment]
can i have your pelispedia.py please?
:-)

**Comment by xxcriticxx** [comment]
@dracosc it stock ver that comes with plugin

**Comment by xxcriticxx** [comment]
young and hungry s03 mostly HQ s04 mostly HD
baby daddy i have HD episodes

**Comment by cabban** [comment]
@xxcriticxx not me :-(

**Comment by xxcriticxx** [comment]
sorry to hear that my friend. did you try to install older version?

**Comment by cabban** [comment]
@xxcriticxx mate which version do u use now?

**Comment by cabban** [comment]
@xxcriticxx mate which version do u use now?

**Comment by cabban** [comment]
So that works for u . Not for me :-(

**Comment by xxcriticxx** [comment]
it may be blocked by your isp or your country

**Comment by cabban** [comment]
but i had been able to get many hd hosters, but not now. only sometimes on
movies

2016-07-13 22:16 GMT+02:00 xxcriticxx <notifications@github.com>:

**Comment by xxcriticxx** [comment]
here is the explanation

http://koditips.com/exodus-no-hd-streams-help-information/

**Comment by cabban** [comment]
Both exodus and salts works but not the best addon specto :-(

**Comment by mrknow** [comment]
fixed
